### DISCUSSION TITLE
TS1C Cordless Soldering Station from Miniware (new product)

### DISCUSSION THREAD

**Discussion by ia** [Ralim/IronOS#1736] [General]
# **DISCLAIMER**: **I'm not affiliated commercially or in any other financial way with companies, organizations nor people mentioned below**.

So in case if someone may be interested but haven't heard yet, just letting community to know that there is a new soldering product from Miniware which is already available to buy.

# TS1C Cordless Soldering Station by Miniware

## Links
[Product page](https://www.miniware.com.cn/product/ts1c-cordless-soldering-station/) 
[Official buy link](https://www.aliexpress.us/item/3256805584218940.html) (provided on product page)
[Official forum topic](http://minidso.com/forum.php?mod=viewthread&tid=5546)
[User Manual (pdf)](https://www.minidso.com/forumdownload/TS1CUserManaul.CN.EN.RU.V1.0.pdf)
[Video Review from Miniware](https://www.youtube.com/watch?v=tOTg-GDXXLM)
[Video Review by SDG Electronics](https://www.youtube.com/watch?v=TkYBXhdQ78Q)
[Video Review by Schematix](https://www.youtube.com/watch?v=rIzO-hix_sM)

## Specifications
Some details based on youtube reviews:
- FCC ID: [2ATIFTS1C-S](https://fccid.io/2ATIFTS1C-S)
- the base has [CH579](https://wch-ic.com/products/CH579.html) MCU which is based on ARM Cortex M0 with integrated BLE 4.2 stack for communication between the base & the iron
- power type:
  - usb-c power delivery 20V/45W for the base
  - the iron has internal supercapacitor (LIC1840Q3R8757 / 750F) which is charged from the base to work remotely, but it has usb-c port so it can get power in a traditional way as well (in the case of need/emergency)
- UI/controls:
  - the base - two-colored (yellow for status bar & blue for main info) OLED 128x64, two buttons and rotary dial
  - the iron - one button for boost mode, one status led
- capacitor time:
  - soldering time without a charge: ~7 minutes
  - charging time to full capacity: ~5 and a half minutes
- tip type: 3.5 audio jack (TS80/P-like compatible)

# IronOS support
## Is it possible (theoretically at least) to see IronOS support for this hardware in the future some day?

**Comment by Ralim** [comment]
I saw this launch as well.
In _theory_ yes support could be added; there is nothing here we couldn't support (Given time).
In practice it would require a bunch of reverse engineering their hardware and testing as Miniware is no longer open source these days.
